docs/howto/openstack/neutron/vpnaas.md

```diff
@@ -0,0 +1,586 @@
+    | Name                     | vpn-conn-to-fra1                                   |
+    | Peer Address             | 198.51.100.50                                      |
+    | Peer CIDRs               |                                                    |
+    | Peer Endpoint Group ID   | 2e627315-02f0-4d68-8683-14230b166060               |
+    | Peer ID                  | 198.51.100.50                                      |
+    | Pre-shared Key           | de12db260ee1b9c0b9e624d910c9a9dbddec13dc24d60332   |
+    | Project                  | 94109c764a754e24ac0f6b01aef82359                   |
+    | Route Mode               | static                                             |
+    | State                    | True                                               |
+    | Status                   | PENDING_CREATE                                     |
+    | VPN Service              | bb1a307d-6f8f-4a0a-83db-7c705403485d               |
+    | dpd                      | {'action': 'hold', 'interval': 30, 'timeout': 120} |
+    | project_id               | 94109c764a754e24ac0f6b01aef82359                   |
+    +--------------------------+----------------------------------------------------+
+    ```
+
+## Viewing VPN connections and getting details
+
+No matter if you use the {{gui}} or the OpenStack CLI, you may at any 
+time list all VPN connections and get relevant details.
+
+=== "{{gui}}"
+    In the vertical pane on the left-hand side of the {{gui}}, expand 
+    the _Networking_ section and then the _VPN Services_ subsection. From 
+    the available options, click _VPN Services_ again. You will see two VPN 
+    connections in the main pane, each from one region to the other.
+        
+    ![Create](assets/vpnaas/shot-06.png)
+        
+    For more information regarding a specific connection, click the 
+    corresponding three-dot icon (right-hand side) and select _View 
+    details_. Then, you can glance over all the details regarding, for 
+    example, the connection status and public IP address.
+        
+    ![Create](assets/vpnaas/shot-07.png)
+=== "OpenStack CLI"
+    You can list all IPSec VPN connections working from any of the two 
+    regions involved. See, for example, the view from `fra1`:
+        
+    ```bash
+    openstack vpn ipsec site connection list
+    ```
+        
+    ```plain
+    +--------------------------+------------------+---------------+--------------------------+--------+
+    | ID                       | Name             | Peer Address  | Authentication Algorithm | Status |
+    +--------------------------+------------------+---------------+--------------------------+--------+
+    | 5f44be31-0588-4f33-883d- | vpn-conn-to-kna1 | 203.0.113.101 | psk                      | ACTIVE |
+    | 9e2d97be55e1             |                  |               |                          |        |
+    +--------------------------+------------------+---------------+--------------------------+--------+
```

Comment:
Looks like this missing a `CLIFF_MAX_WIDTH=100` setting; is this intentional?

Reply:
Um, no, `CLIFF_MAX_WIDTH=100` is actually in both RC files I use.

Reply:
Odd. Perhaps Cliff just doesn't want to ever wrap header cells, and thus the `Authentication Algorithm` cell pushed the line width over the limit.